scripts/example.ts

```diff
@@ -171,19 +210,40 @@ async function main() {
       dryRun,
     });
 
+    // TODO -- this seems like it might be framework specific. Should we search for a folder?
+    // or set it in the config somewhere?
+    const storiesDir = path.resolve(cwd, './stories');
+
+    // TODO -- can we get the options type to return something more specific
```

Comment:
What about
```ts
const frameworks =  ['react', 'angular'] as const
type Frameworks = keyof typeof frameworks;
```

Reply:
I had something more complex in mind that I am still trying to work out. Basically I want

```ts
const option = {
  values: ['react', 'angular'] as const,
}

// This should have type `'react' | 'angular'`.
const value: OptionValue<typeof option>;
```

I haven't quite figured this one out yet.

Reply:
Going to sort this out later, thanks @yannbf